ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -52,6 +52,16 @@ tests:
     test:
     - ref: openshift-qe-node-density
     workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-ingress-perf
+  cron: 0 12 * * 5
```

Comment:
Are we going to stick with 3 nodes testing for ingress perf or do we plan to scale? 
Not sure if this is relevant thread, but could you ptal https://redhat-internal.slack.com/archives/GTHU29259/p1688584111328949

Reply:
For now, we can stick with 3 node testing.

We have baselined with different instance sizes and node-labeling.
which I'd like to tackle in a different PR & figure out ci-operator docs on how to achieve

Reply:
Harsha, if you see other Rosa examples, you will see various env variables on how to specify instance sizes, zones etc.. I haven't tried node labeling yet.

Reply:
Yes, re: Instance Sizes I observed in other ROSA examples.
Resolving this conversation for now.